packages/lesswrong/server/fmCrosspost/denormalizedFields.ts

```diff
@@ -0,0 +1,31 @@
+import { hasStringParam, hasBooleanParam } from "./validationHelpers";
+import { pick } from "underscore";
```

Comment:
```suggestion
import pick from "lodash/pick";
```

Underscore is deprecated